package.json

```diff
@@ -3,9 +3,9 @@
   "version": "7.1.0",
   "description": "babel module loader for webpack",
   "files": [
-    "lib"
+    "src"
   ],
-  "main": "lib/index.js",
```

Comment:
It looks like using a proper build tarball fixed my issues and everything is working again. Thanks for that.